sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CopyAuthorization.java

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.azure.core.util.serializer.SerializerAdapter;
+import com.azure.core.util.serializer.SerializerEncoding;
+
+import java.io.IOException;
+
+/**
+ * The CopyAuthorization model.
+ */
+@Fluent
+public final class CopyAuthorization {
+    private static final SerializerAdapter SERIALIZER = new JacksonAdapter();
+
+    CopyAuthorization() {
+        super();
+    }
+
+    /*
+     * Model identifier.
+     */
+    private String modelId;
+
+    /*
+     * Token claim used to authorize the request.
+     */
+    private String accessToken;
+
+    /*
+     * Resource Identifier.
+     */
+    private String resourceId;
+
+    /*
+     * Region of the resource.
+     */
+    private String region;
+
+    /*
+     * The time when the access token expires. The date is represented as the
+     * number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.
+     */
+    private long expiresOn;
+
+    /**
+     * Create a CopyAuthorization object
+     *
+     * @param modelId The model identifier
+     * @param accessToken The token used to authorize the request
+     * @param resourceId The resource identifier
+     * @param region The region of the resource
+     * @param expiresOn The expiry time of the token
+     */
+    public CopyAuthorization(final String modelId, final String accessToken, final String resourceId,
+        final String region, final long expiresOn) {
+        this.modelId = modelId;
+        this.accessToken = accessToken;
+        this.resourceId = resourceId;
+        this.region = region;
+        this.expiresOn = expiresOn;
+    }
+
+    /**
+     * Get the modelId property.
+     *
+     * @return the modelId value.
+     */
+    public String getModelId() {
+        return this.modelId;
+    }
+
+    /**
+     * Get the token claim used to authorize the request.
+     *
+     * @return the accessToken value.
+     */
+    public String getAccessToken() {
+        return this.accessToken;
+    }
+
+    /**
+     * Get the time when the access token expires. The date is represented as the
+     * number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.
+     *
+     * @return the expirationDateTimeTicks value.
+     */
+    public long getExpiresOn() {
+        return this.expiresOn;
+    }
+
+    /**
+     * Get the Azure Resource Id of the target Form Recognizer resource
+     * where the model will be copied to.
+     *
+     * @return the resourceId value.
+     */
+    public String getResourceId() {
+        return resourceId;
+    }
+
+    /**
+     * Get the location of the target Form Recognizer resource.
+     *
+     * @return the resourceRegion value.
```

Comment:
Here you say `resourceRegion` but below shows `region`. Is that right?

Reply:
I'm not aware if you and @maririos discussed this when creating the copy API, but python has it as "resourceRegion" in our dict, and it seems that Java and .NET have it as region. I'm pretty sure this is not an issue to have python different, just wanted to make sure.

Reply:
sorry @kristapratico I deleted the previous comment before realizing you replied to it since it was wrong. Feel free to ignore